test/fixtures/malformed/web-frame.md

```diff
@@ -109,6 +109,43 @@ this limitation.
 
 ### `webFrame.getResourceUsage()`
 
+Returns `Object`:
```

Comment:
All sorted, had to update some other tests elsewhere after I realized that the method description was including the "Returns" line in it